recipes/asionet/all/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, tools
+
+class ASIOnetConan(ConanFile):
+    name = "asionet"
+    version = "1.5.0"
```

Comment:
Please remove the version from the python script so that you can reuse it for other versions in the future.